bundles/org.openhab.binding.evcc/README.md

```diff
@@ -38,50 +38,66 @@ Default value for _refreshInterval_ is 60 seconds.
 Those channels exist only once.
 Please note that some of them are only available when evcc is properly configured.
 
-| Channel                    | Type                 | Read/Write | Description                                                                                                  |
-|----------------------------|----------------------|------------|--------------------------------------------------------------------------------------------------------------|
-| general#batteryCapacity    | Number:Energy        | R          | Capacity of (home) battery.                                                                                  |
-| general#batteryPower       | Number:Power         | R          | Current power from battery.                                                                                  |
-| general#batterySoC         | Number:Dimensionless | R          | Current State of Charge of battery.                                                                          |
-| general#batteryPrioritySoC | Number:Dimensionless | RW         | State of State of Charge for which the battery has priority over charging the ev when charging mode is "pv". |
-| general#gridPower          | Number:Power         | R          | Current power from grid (negative means feed-in)                                                             |
-| general#homePower          | Number:Power         | R          | Current power taken by home.                                                                                 |
-| general#pvPower            | Number:Power         | R          | Current power from photovoltaik.                                                                             |
+| Channel                         | Type                 | Read/Write | Description                                                                                                       |
```

Comment:
Can you please fix the table formatting? On line is out of the bounds.

You may use http://markdowntable.com/.